### PR TITLE
Superuser bool away lib

### PR DESCRIPTION
**Pull request by jelly**

### CODE REVIEW

**Review by martinpitt — APPROVED** [review]
Thanks! I feel like these could be squashed, but no strong opinion.

File: pkg/storaged/client.js [hunk 1]
```diff
@@ -1052,7 +1052,7 @@ client.mount_at = (block, target) => {
     if (entry)
         return cockpit.script('set -e; mkdir -p "$2"; mount "$1" "$2" -o "$3"',
                               [utils.decode_filename(block.Device), target, utils.get_block_mntopts(entry[1])],
-                              { superuser: true, err: "message" });
+                              { superuser: "require", err: "message" });
```
Comment (cockpituous) [inline pkg/storaged/client.js:1055]: This added line is not executed by any test.

File: pkg/storaged/client.js [hunk 2]
```diff
@@ -1201,7 +1201,7 @@ function legacy_vdo_overlay() {
     function cmd(args) {
         return cockpit.spawn(["vdo"].concat(args),
                              {
-                                 superuser: true,
+                                 superuser: "require",
```
Comment (cockpituous) [inline pkg/storaged/client.js:1204]: This added line is not executed by any test.

File: pkg/storaged/client.js [hunk 3]
```diff
@@ -1460,7 +1460,7 @@ function stratis2_start() {
                     return Promise.reject(new Error("stratisd too old, need at least version 2.4"));
 
                 client.stratis_store_passphrase = (desc, passphrase) => {
-                    return python.spawn(stratis2_set_key_py, [desc], { superuser: true })
+                    return python.spawn(stratis2_set_key_py, [desc], { superuser: "require" })
```
Comment (cockpituous) [inline pkg/storaged/client.js:1463]: This added line is not executed by any test.

File: pkg/storaged/legacy-vdo/legacy-vdo.jsx [hunk 4]
```diff
@@ -184,7 +184,7 @@ class VDODetails extends React.Component {
         }
 
         if (path)
-            this.poll_process = cockpit.spawn([client.legacy_vdo_overlay.python, "--", "-", path], { superuser: true })
+            this.poll_process = cockpit.spawn([client.legacy_vdo_overlay.python, "--", "-", path], { superuser: "require" })
```
Comment (cockpituous) [inline pkg/storaged/legacy-vdo/legacy-vdo.jsx:187]: This added line is not executed by any test.

File: pkg/storaged/legacy-vdo/legacy-vdo.jsx [hunk 5]
```diff
@@ -263,7 +263,7 @@ class VDODetails extends React.Component {
                                 if (block && block.IdUsage == "filesystem")
                                     return cockpit.spawn(["fsadm", "resize",
                                         decode_filename(block.Device)],
-                                                         { superuser: true, err: "message" });
+                                                         { superuser: "require", err: "message" });
```
Comment (cockpituous) [inline pkg/storaged/legacy-vdo/legacy-vdo.jsx:266]: This added line is not executed by any test.

File: pkg/systemd/overview-cards/insights.jsx [hunk 6]
```diff
@@ -113,7 +113,7 @@ export class InsightsStatus extends React.Component {
                 this.pollster = null;
             }
             if (data)
-                this.pollster = cockpit.script(insights_poll_hack_sh, [], { superuser: true });
+                this.pollster = cockpit.script(insights_poll_hack_sh, [], { superuser: "require" });
```
Comment (cockpituous) [inline pkg/systemd/overview-cards/insights.jsx:116]: This added line is not executed by any test.

**Pull request merged**